web/src/notifications.js

```diff
@@ -278,7 +278,8 @@ function get_notification_recipients(message, content, title) {
         ) {
             // Then count how many people are in the conversation and summarize
             // by saying the conversation is with "you and [number] other people"
-            recipients = recipients.replaceAll(/[^,]/g, "").length + " other people";
+            const recipient_count = recipients.replaceAll(/[^,]/g, "").length;
+            recipients = $t({defaultMessage: "{recipient_count} other people"}, {recipient_count});
```

Comment:
This string probably needs to use the ICU plurals syntax: https://zulip.readthedocs.io/en/latest/translating/internationalization.html#frontend-translations

We use it in a few places in the code base; here we know `people` could be 2 or 3+ and in some languages that requires different case.

Reply:
Good catch, thanks! I've added a string like this:
```javascript
defaultMessage: "{recipient_count} other {recipient_count, plural, one {# person} other {# people}}"
```

We are aware that in English, it's typically just "people," so I used this as an example for translators. However, if it's necessary to convey that there will always be at least two people, perhaps I should write it like this:
```javascript
defaultMessage: "{recipient_count} other {recipient_count, plural, two {# people} other {# people}}"
```

Reply:
Not sure which string option is the best.

Reply:
I think the two option is probably best.